consensus/polybft/consensus_runtime_test.go

```diff
@@ -782,6 +783,149 @@ func TestConsensusRuntime_calculateStateTxsInput_SecondEpoch(t *testing.T) {
 	polybftBackendMock.AssertExpectations(t)
 }
 
+func TestConsensusRuntime_generateSyncValidatorsDataTxInputRemoved(t *testing.T) {
+	t.Parallel()
+
+	const (
+		epoch           = 1
+		epochStartBlock = 1
+	)
+
+	validators := validator.NewTestValidatorsWithAliases(t, []string{"A", "B", "C", "D", "E"})
+	accSet := validators.GetPublicIdentities()
+
+	blockNumber := uint64(10)
+	parentHash := types.BytesToHash(big.NewInt(int64(blockNumber)).Bytes())
+	removedVals := bitmap.Bitmap{}
+	removedVals.Set(2)
+
+	dummySignature := [64]byte{}
+	extraData := Extra{
+		Validators: &validator.ValidatorSetDelta{
+			Removed: removedVals,
+		},
+		Parent:     &Signature{Bitmap: nil, AggregatedSignature: dummySignature[:]},
+		Committed:  &Signature{Bitmap: nil, AggregatedSignature: dummySignature[:]},
+		Checkpoint: &CheckpointData{EpochNumber: epoch},
+	}
+
+	lastBuiltBlock := &types.Header{
+		Number:     blockNumber,
+		ParentHash: parentHash,
+		ExtraData:  extraData.MarshalRLPTo(nil),
+		GasLimit:   types.StateTransactionGasLimit,
+	}
+
+	polybftBackendMock := new(polybftBackendMock)
+	polybftBackendMock.On("GetValidators", mock.Anything, mock.Anything).
+		Return(accSet)
+
+	config := &runtimeConfig{
+		polybftBackend: polybftBackendMock,
+		Key:            validators.GetValidator("A").Key(),
+	}
+
+	consensusRuntime := &consensusRuntime{
+		config: config,
+		epoch: &epochMetadata{
+			Number:            epoch,
+			Validators:        accSet,
+			FirstBlockInEpoch: epochStartBlock,
+		},
+		lastBuiltBlock: lastBuiltBlock,
+		logger:         hclog.NewNullLogger(),
+	}
+
+	valSet := validator.NewValidatorSet(consensusRuntime.epoch.Validators.Copy(), consensusRuntime.logger)
+	valSetAccounts := valSet.Accounts()
+
+	parentIbftExtraData, err := GetIbftExtra(lastBuiltBlock.ExtraData)
+	assert.NoError(t, err)
+
+	validatorsDelta := parentIbftExtraData.Validators
+
+	expectedUpdatedValidatorsPower := extractUpdatedValidatorsVotingPower(validatorsDelta, valSetAccounts, accSet)
```

Comment:
instead of using additional function, can't you hardcode the expectedUpdatedValidatorsPower